sdk/java-sdk/src/main/java/org/lightswitch/sdk/user/LightSwitchUser.java

```diff
@@ -0,0 +1,38 @@
+package org.lightswitch.sdk.user;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class LightSwitchUser {
+
+    private final String userId;
+    private final Map<String, String> attributes;
```

Comment:
It looks like `LightSwitchUser` serves a different purpose than the backend's `User`, where is this data stored?